app/models/email_digests/linkedin_profiles.rb

```diff
@@ -0,0 +1,20 @@
+class EmailDigests::LinkedinProfiles
+  def send_weekly_digest
+    # return unless Time.current.monday?
```

Comment:
What's the rationale behind this comment?

Reply:
Removed it as it is not required